mandatory/3-function-output.js

```diff
@@ -11,6 +11,7 @@ function combine2Words(word1, word2) {
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return( `${firstWord} ${secondWord} ${thirdWord}`)
```

Comment:
Consider this solution: return firstWord.concat(" ", secondWord, " ", thirdWord);

Also there are two questions before you should write the answer too. You can write answer like this // I created a descriptive word to...